src/components/atoms/FormModal.tsx

```diff
@@ -1,16 +1,115 @@
+import { COLOR } from 'consts'
 import React, { ReactElement } from 'react'
-import { Modal, ModalProps, SafeAreaView } from 'react-native'
+import {
+  Modal,
+  ModalProps,
+  StyleSheet,
+  TouchableOpacity,
+  View,
+} from 'react-native'
+import FormText from './FormText'
 
-const FormModal = (props: ModalProps): ReactElement => {
-  const { children, ...rest } = props
+const FormModal = (
+  props: ModalProps & {
+    title: string
+    message: string
+    positive?: string
+    positiveCallback?: () => void
+    negative?: string
+    negativeCallback?: () => void
```

Comment:
just add this to ModalProps. Seems generic. Also, would be better to make a type for modal button type `(text: string, callback: () => void)`, and make `positive?: ModalButton, negative?: ModalButton`